doc/API/centreon-api-v2.yaml

```diff
@@ -2332,6 +2357,38 @@ components:
         allOf:
           - $ref: '#/components/schemas/Check.Service'
           - $ref: '#/components/schemas/Bulk.Resource.Id'
+    Monitoring.Resource.Disacknowledge.Host:
+      type: object
+      properties:
+        type:
+          type: string
+          description: "Type of the resource"
+          example: "service|host"
+        id:
+          type: integer
+          format: int32
+          description: "ID of the resource"
+          example: 12
+    Monitoring.Resource.Disacknowledge.Service:
+      type: object
+      properties:
+        type:
+          type: string
+          description: "Type of the resource"
+          example: "service|host"
+        id:
```

Comment:
could you use resource_id and parent_resource_id ?
By referencing '#/components/schemas/Bulk.Resource.Id'

Reply:
not sure what you mean, the structure of the request has already been defined and approved and is specific for this use case, and I need the type so that I can distinguish host from service.

Reply:
because it will not be consistent with other bulk endpoints : 
- https://api-documentation.centreon.com/centreon-web/centreon-api-v2.html#tag/Acknowledgement/paths/~1monitoring~1hosts~1acknowledgements/post
- https://api-documentation.centreon.com/centreon-web/centreon-api-v2.html#tag/Acknowledgement/paths/~1monitoring~1services~1acknowledgements/post

Where type is managed by the route path